calm/control-example/one-node-wonder.json

```diff
@@ -8,21 +8,16 @@
       "node-type": "system",
       "name": "Example System",
       "description": "Example System",
-      "domains": {
-        "cbom": [
-          {
-            "unique-id": "ci-arch-001",
-            "description": "Evidence of Pre Production Review",
-            "requirement-schema": "control-specification.json",
-            "requirement-instance": "control-instance.json",
-            "evidence": [
+      "controls": {
+        "cbom": {
+            "base-path": ".",
+            "description": "Control requirements for delivering patterns",
+            "requirements": [
               {
-                "evidence-link": "https://evidence.com/12345",
-                "evidence-id": "12345"
+                "$ref": "pre-prod-review-configuration.json"
```

Comment:
Shouldn't this be a link to the pre-prod-review-specification.json rather than the configuration?